### ISSUE TITLE
The ChatGPT 3.5 Turbo model should be marked as a completions model

### ISSUE THREAD

**Issue by TimRivoli** [lambiengcode/compose-chatgpt-kotlin-android-chatbot#12]
First off, very nice work!
Second, GPT 3.5 Turbo is a completions model so I made this fix to make the app work for me.
gpt35Turbo("gpt-3.5-turbo", 4096, isChatCompletion = true)

**Comment by lambiengcode** [comment]
yes,  it's my mistake, thank you @TimRivoli. I'll fix it tomorrow.

**Comment by 4niso** [comment]
Thank you @TimRivoli, this work form me.
